Gordon360/Controllers/NewsController.cs

```diff
@@ -198,5 +199,26 @@ public ActionResult<StudentNewsViewModel> EditPosting(int newsID, [FromBody] Stu
             var result = _newsService.EditPosting(newsID, studentNewsEdit);
             return Ok(result);
         }
+
+        /// <summary>
+        ///  Approve a news posting in the database
+        /// </summary>
+        /// <param name="newsID">The id of the news item to approve</param>
+        /// <returns>The approved news item</returns>
+        /// <remarks>The news item must not be expired and must be unapproved</remarks>
+        [HttpPut]
+        [Route("approvalStatus")]
+        // only SNAdmin is authorized to use this route
+        // Can't use SYB UPDATE operation, because the author is not authorized to approve the post
+        public ActionResult<StudentNewsViewModel> ApprovePosting([FromBody] int newsID)
```

Comment:
```suggestion
        [Route("{newsID}/accepted")]
        // only SNAdmin is authorized to use this route
        // Can't use SYB UPDATE operation, because the author is not authorized to approve the post
        public ActionResult<StudentNewsViewModel> ApprovePosting([FromBody] bool newStatusAccepted)
```
Restful interfaces should reference specific resources. Wherever possible, the routes should be structured to reference a particular item and its property rather than a workflow.